JS/Lesson15/sinan/script.js

```diff
@@ -175,5 +316,32 @@ whichMonthName(10);
 whichMonthName(2);
 
 // 14. Write a program that asks the user to enter a grade letter (A, B, C, D, or F) and displays a message indicating whether the grade is passing (A, B, C) or failing (D, F). Use a switch statement to handle the different cases.
+function gradeTest (point){
+	switch(point){
+		case 'A': 
+		case 'B':
+		case 'C':
+			console.log('The grade is passing');
+			break;
+		case 'D': 
+		case 'F':
+			console.log ('The grade is falling');
+			break;
+	}
+}
+gradeTest('a'.toUpperCase());
+
 
 // 15. Write a program that asks the user to enter their age, and then calculates and displays their age in dog years (where one human year is equivalent to seven dog years). If the user enters a negative number or zero, display an error message.
+
+function dogYear(dogAge) {
+	let humanAge = dogAge * 7;
```

Comment:
Move this statement to the `else` condition. You'd run into issues if the value passed to the function is not a positive number!